tests/framework/utils/k8s_helper.go

```diff
@@ -1701,9 +1703,18 @@ func (k8sh *K8sHelper) WaitForLabeledDeploymentsToBeReadyWithRetries(label, name
 }
 
 func (k8sh *K8sHelper) WaitForCronJob(name, namespace string) error {
+	k8sVersion, err := k8sutil.GetK8SVersion(k8sh.Clientset)
+	if err != nil {
+		return errors.Wrap(err, "failed to fetch get k8s version")
```

Comment:
```suggestion
		return errors.Wrap(err, "failed to fetch k8s version")
```
I think it is coming from my last change https://github.com/rook/rook/blob/5f30d538a416e3e2d4569845c4d77c9cd384859d/pkg/operator/ceph/cluster/crash/reconcile.go#L282 can you change here too. thanks